File: go/pull-with-push-override/fs/foo-sfl-cfg-adapter.go
/*
 * Copyright © 2022 Paulo Villela. All rights reserved.
 * Use of this source code is governed by the MIT license
 * that can be found in the LICENSE file.
 */

package fs

import (
	"github.com/pvillela/module-config/go/pull-with-push-override/config"
)

func FooSflCfgAdapter(appCfg config.AppCfgInfo) FooSflCfgInfo {
	return FooSflCfgInfo{
		X: appCfg.X,
	}
}

File: rs/pushtovar/src/fs/foo_sfl_cfg_adapter.rs
use crate::config::app_cfg_info::AppCfgInfo;

use super::foo_sfl::FooSflCfgInfo;

pub fn foo_sfl_cfg_adapter(app_cfg: &AppCfgInfo) -> FooSflCfgInfo {
    FooSflCfgInfo {
        x: app_cfg.x.clone(),
    }
}
